gitops/predicates_test.go

```diff
@@ -87,18 +101,26 @@ var _ = Describe("Predicates", Ordered, func() {
 	})
 
 	AfterAll(func() {
```

Comment:
`bindingMissingStatus` should be deleted together here. Others look good to me. Thanks!